### ISSUE TITLE
Notebooks are broken

### ISSUE THREAD

**Issue by aarchiba** [nanograv/PINT#1486]
Running `make notebooks` fails; this may be impeding the readthedocs build. Our CI should have caught this and reported failure. 

In some cases the failure is a result of using out-of-date code; in others I'm not sure what happened. There are also some that don't result in correct output.

**Comment by aarchiba** [comment]
We haven't been running the notebook tests:
```
Run tox -e notebooks
notebooks: skipped because could not find python interpreter with spec(s): python3.8
  notebooks: SKIP (0.76 seconds)
  evaluation failed :( (1.04 seconds)
```
This is fixable in `tox.ini`.

**Comment by dlakaplan** [comment]
@aarchiba : was this fixed by #1487?

**Comment by abhisrkckl** [comment]
This issue no longer exists. Closing.
